Borysov/quiz-boot/src/main/java/com/quiz/borysov/quizboot/config/QuizConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package com.quiz.borysov.quizboot.config;
+
+
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.orm.hibernate5.HibernateTransactionManager;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.sql.DataSource;
+
+@Configuration
+public class QuizConfiguration {
+}
```

Comment:
Why you need this empty configuration class?

Reply:
i will delete it